app/views/spree/admin/orders/customer_details/_address_form.html.haml

```diff
@@ -0,0 +1,45 @@
+- if use_billing
+  .field{style: "position: absolute;margin-top: -15px;right: 0;"}
+    %span
+      = check_box_tag 'order[use_billing]', '1', (!(@order.bill_address.empty? && @order.ship_address.empty?) && @order.bill_address.same_as?(@order.ship_address))
+      = label_tag 'order[use_billing]', Spree.t(:use_billing_address)
+
+- is_shipping_address = name == Spree.t(:shipping_address)
+- s_or_b = is_shipping_address ? 's' : 'b'
```

Comment:
I'm not a big fan of these single-letter bits, but not a big deal.

Reply:
I think it's terrible. I am bringing this from spree. I just dont want to fix too many things as I bring stuff from spree, so that I dont risk breaking it.